catalog/core/catalog-core-api/src/main/java/ddf/catalog/data/Metacard.java

```diff
@@ -88,6 +90,11 @@
     public static final String ID = "id";
 
     /**
+     * {@link Attribute} name for accessing the tags of the {@link Metacard}. <br/>
+     */
+    public static final String TAGS = "tags";
```

Comment:
Updated tags to metacard-tags. I didn't use the dot notation since that isn't used anywhere else in the Metacard class. Let me know if you think it should really be metacard.tags